index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <nav class="main-nav">
+            <div class="left-bar">
+                <p class="time">9:41</p>
+                <p class="name">Oscar Byiringiro</p>
+            </div>
+            <div class="right-bar">
+                <div class="top-icons">
+                    <img src="./images/Cellular Connection.png">
+                    <img src="./images/Wifi.png">
+                    <img src="./images/Battery.png">
+                                
+                            
+                </div>
+                <div class="bottom-icons">
+                    <img src="./images/Union.png">
+                        
+                </div>
+                
+                    
+            </div>
+        </nav>
+        <section class="main-body">
+            <div class="container">
+                <div class="intro">
+                    <p>Hey There</p>
+                    <p>I'm Oscar</p>
+                </div>
+                <div class="title">
+                    <p>I'am a software developer</p>
+                </div>
+                <div class="description">
+                    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.</p>
+                </div>
+                <div class="logo">
+                    <img src="./images/twitter.png">
+                    <div class="linkedin">
+                        <img src="./images/Group-1.png">
+                            
+                        <img src="./images/Group.png">
+                    </div>
+                    <img src="./images/hand.png">
+                    <img src="./images/m.png">
+
+                                        
+                                    
+                                                                              
+                </div>
+                <div class="main-image">
+                    <img src="./images/illustration-header-mobile.png">
+                </div>
+            <div>
```

Comment:
- [ ] Great job adding image to replicate the figma design. However, as suggested, please make sure that you avoid using `<img>` tag adding images. Instead, you can use `background-image` property for doing that. For more clarity, please visit [this link](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image). I know you can do that 💪


### **_Your design previously 👇_**
![image](https://user-images.githubusercontent.com/106119105/212499013-34946207-7465-4937-aed2-e1f58c566ca3.png)

### **_Your design now 👇_**
![image](https://user-images.githubusercontent.com/106119105/212499029-c27e04bf-8cd4-4be3-9cd9-284811cb0c8d.png)

### **_Figma design 👇_**
![image](https://user-images.githubusercontent.com/106119105/212499039-f095ffbc-5212-467c-a6a8-45f70a1b44bf.png)